ingestion/src/metadata/ingestion/source/salesforce.py

```diff
@@ -94,7 +94,14 @@ def create(cls, config: dict, metadata_config: dict, ctx: WorkflowContext):
     def column_type(self, column_type: str):
         if column_type in ["ID", "PHONE", "CURRENCY"]:
             type = "INT"
-        elif column_type in ["REFERENCE", "PICKLIST", "TEXTAREA", "ADDRESS", "URL"]:
+        elif column_type in [
+            "REFERENCE",
+            "PICKLIST",
+            "TEXTAREA",
+            "ADDRESS",
+            "URL",
+            "EMAIL",
+        ]:
```

Comment:
Thanks for the changes @codingwithabhi. Now that we are changing this, we could make a couple of improvements to this function:
- it is more performant to return directly in the `if` statement:
```python
if column_type in ...:
     return "INT"
if column_type in ...:
    return "VARCHAR"
return column_type
```
This way we can skip some validations.

- The other point is about increasing the lookup speed. Instead of `... in [a,b,c]`, use a set `... in {a, b, c}`.
Thanks!